libs/rpc.red

```diff
@@ -0,0 +1,34 @@
+Red []
+
+context [
+
+	headers: compose [
+		Content-Type: "application/json"
+		Accept: "application/json"
+		User-Agent: (
+			form reduce [
+				{Red Wallet version}
+				read %version.red
```

Comment:
The executable must be along with the %version.red file. Can we use macro here? so we get the version in compiling time.

Reply:
:+1: Will do in the evening, thx!